components/elasticsearch/scripts/create.py

```diff
@@ -17,15 +17,18 @@
 CONFIG_PATH = "components/elasticsearch/config"
 ES_SERVICE_NAME = 'elasticsearch'
 
-ctx_properties = utils.ctx_factory.create('elasticsearch')
+UPGRADE_DUMP_PATH = '/tmp/es_upgrade_dump/'
```

Comment:
These are script constants. I can make them start with '_' if that's what you mean